### PR TITLE
Interactive list feature

### PR DESCRIPTION
**Pull request by thasipablo**
- Add a new JavaScript file and import it as a module.
- Implement a function for the "Clear all completed" tasks.
- Store the updated array of items in local storage.

### CODE REVIEW

File: src/statusFunctions.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      tasks.splice(index, 1);
+    }
+  });
+
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+
+  saveTasks();
+};
```
Comment (franclobo) [inline src/statusFunctions.js]: - [ ] The `Clear all completed` button works as expected but there is a requirement to be completed to pass to the next milestone, please 🙏🏽 use the `filter` method 👇🏽 

> Implement a function for the "Clear all completed" button (use filter() method).

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @thasipablo,

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- App works as expected ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/statusFunctions.js (hunk 1 above)
Comment (thasipablo) [reply src/statusFunctions.js]: thank you for this feed back.

**Review by Munsa1 — APPROVED** [review]
Hi @thasipablo , 
Approved 1

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Used Filter Method As Requested :heavy_check_mark: 
App Works As Expected :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**